src/scss/_pager.scss

```diff
@@ -1,85 +1,81 @@
-.jazz-pager {
-  display: inline-block;
+.jazz-pager > ul {
+  font-family: jazz-theme(body);
```

Comment:
I think this reference should be "body-font" instead of just "body".  Alternatively, you could change it to be "pager-item-font" if you want the developer to be able to control the pager font separately from the body font.